lineapy/api/api.py

```diff
@@ -359,32 +377,88 @@ def to_pipeline(
     framework: str = "SCRIPT",
     pipeline_name: Optional[str] = None,
     dependencies: TaskGraphEdge = {},
-    pipeline_dag_config: Optional[AirflowDagConfig] = {},
     output_dir: str = ".",
+    input_parameters: List[str] = [],
+    reuse_pre_computed_artifacts: List[str] = [],
```

Comment:
The signature here is different from `get_function`

Reply:
We want to retain this type signature (`List[str]`) for `reuse_pre_computed_artifacts` since `to_pipeline()` has never supported artifact versioning so far. We want to open a separate ticket for adding artifact versioning support to `to_pipeline()` API.

Discussed this w/ @mingjerli in person and we are on the same page.

Reply:
Yes, we've created a ticket https://linea.atlassian.net/browse/LIN-593  to track this.